src/Apple/Provider.php

```diff
@@ -170,6 +220,25 @@ public static function verify($jwt)
         throw new InvalidStateException('Invalid JWT Signature');
     }
 
+    /**
+     * Verify Apple jwt via static function.
+     *
+     * @param string $jwt
+     *
+     * @return bool
+     *
+     * @see https://appleid.apple.com/auth/keys
+     */
+    public static function verify($jwt)
+    {
+        return (new self(
+            new Request(),
+            config('services.apple.client_id'),
+            config('services.apple.client_secret'),
```

Comment:
Will this method still work if the secret is dervied?

Reply:
Yes it will work. I had to do that (create new object) because this function is called statically (don't know why) and it was necessary to call other non static functions.